### ISSUE TITLE
ClaimReview needs field for URL of claims / integration of new Claim type

### ISSUE THREAD

**Issue by BillAdairDuke** [schemaorg/schemaorg#1828]
After discussing the use of ClaimReview with fact-checkers, there is confusion among the fact-checkers about how to populate the field for SameAs field. Some fact-checkers have been using for the URL of speeches or articles where the claim appeared to provide evidence and details of the claim; others have been using it for the home page/knowledge base article on the person or entity that made the statement.

Both have value for search engines and other uses. It’s valuable to be able to see a knowledge base article on the person or entity being checked; it’s also valuable to be able to see where a false claim was published, particularly the article or speech where the claim originated.

**The solution:** We propose that:
sameAs will continue to be used for the entity or speaker that made the the claim (we called it the “who to blame” field). This field would typically be the home page of the person making the claim, or their Wikipedia page.

We will add a field(s), under ClaimReview, called claimUrlOriginal of type URL to designate the URL of the origin of a claim such as a speech or article; fact-checkers could include additional URLs (claimUrl) of type URL if the claim has been repeated in other publications.

For example, here is a fact-check by PolitiFact of a claim by New Century Times. The SameAs field is the website’s URL. The claimURLOriginal field would be the article that contained the claim that was fact-checked. That claim also appeared on Vote.us.org, which is listed as a claimURL.

```javascript
{
  "@context": "http://schema.org",
  "@type": "ClaimReview",
  "datePublished": "2018-01-05",
  "url": "http://www.politifact.com/punditfact/statements/2018/jan/05/new-century-times/white-house-staffers-removed-job-subject-false/",
  "claimReviewed": "Breaking: FBI just raided the White House, 6 people thrown out.",
  "claimUrlOriginal": "http://info.nct.news/2017/12/30/breaking-fbi-just-raided-the-white-house-high-ranking-officials-thrown-out/",
  "claimUrl": ["https://www.vote.us.org/memo/thread/3419/fbi-just-raided-the-white-house-6-people-thrown-out/"],
  "author":
  {
    "@type": "Organization",
    "name": "PolitiFact",
    "url": "http://www.politifact.com"
  },
  "itemReviewed":
  {
    "@type": "CreativeWork",
    "author":
    {
      "@type": "Person",
      "name": "New Century Times",
      "jobTitle": "website",
      "sameAs": "http://newcenturytimes.com/"
    },
    "datePublished": "2017-12-30"
  },
  "reviewRating":
  {
    "@type": "Rating",
    "ratingValue": "2",
    "bestRating": "7",
    "worstRating": "1",
    "alternateName": "False"
  }
}
```
Edited 2018-01-23 by @danbri to highlight key terms, omit HTML script tag, improve formatting.

**Comment by danbri** [comment]
Thanks @BillAdairDuke, good to get this discussion moving.  I agree there's an issue here and it is important to find a pragmatic solution.

One quick clarification about sameAs, ... it has potentially several roles in that it can be used on each typed entity, whether we're talking about the (Claim)Review, the Person, or a CreativeWork like NewsArticle.  Conceivably there could be several sameAs properties in a single description.

Regarding "claimUrlOriginal", does this always indicate the URL of the itemReviewed? If so we could potentially make it an "url" property of that entity.

Some of the same issues might also be addressed by introducing a new type that more explicitly represents the actual *claim* under discussion. So if the claim is that "The moon is made of green cheese", right now Schema.org only represents that fairly indirectly; we talk about a review of that claim and then kind of casually mention the claim in passing via the claimReviewed property. In other words our current model is very much oriented around a bottom up way of dealing with claims as they occur, rather than attempting to normalize them into abstract claims that are unrelated to their occurance in real life. My impression is that by introducing claimUrl we would be getting into very similar territory. Are there factchecking-oriented sites that try identify claims quite separate from their specific occurrences?

**Comment by BillAdairDuke** [comment]
Thanks for the feedback, Dan. Answers below, with a request for clarification.

_Regarding "claimUrlOriginal", does this always indicate the URL of the itemReviewed? If so we could potentially make it an "url" property of that entity._

Yes, “claimURLOriginal” would always indicate the URL of the statement that is reviewed. In some cases that would be the URL of a text or speech. It could also be a news article that contains the claim in a quote of the person.

**Some of the same issues might also be addressed by introducing a new type that more explicitly represents the actual claim under discussion. So if the claim is that "The moon is made of green cheese", right now Schema.org only represents that fairly indirectly; we talk about a review of that claim and then kind of casually mention the claim in passing via the claimReviewed property. In other words our current model is very much oriented around a bottom up way of dealing with claims as they occur, rather than attempting to normalize them into abstract claims that are unrelated to their occurance in real life. My impression is that by introducing claimUrl we would be getting into very similar territory. Are there factchecking-oriented sites that try identify claims quite separate from their specific occurrences?**

I’m not clear what you mean in that the schema represents the claim “fairly indirectly.” In practice, it seems to me that it’s clear that “itemReviewed” is what is being fact-checked. I don't think there's been any confusion about that by users.

I’m also not sure what you mean by “rather than attempting to normalize them into abstract claims that are unrelated to their occurance in real life.”

_Are there factchecking-oriented sites that try identify claims quite separate from their specific occurrences?_

Also not sure what you mean here. If you mean, "Do fact-checkers distinguish repeated claims?" - Practices vary. Some publishers, such as the Washington Post, will put multiple similar statements by different people at the top of an article, although not in any structured way. Others, such as PolitiFact and Snopes, will just mention that the claim or a similar one has been repeated by others. This is particularly clear with talking points that get repeated by many politicians and publications.

**Comment by jkosslyn** [comment]
Dan -- basically, we're trying to make a distinction between the entity responsible for a claim and places where the claim has appeared. Currently we have the former but not the latter.

**Comment by cong-yu** [comment]
Hi All,

Here is my take on this.

- A claim can be a tweet, a person's quote in a rally, an article, etc.
- Each claim can be repeated in many places and may also have an origin, where the claim originally appeared.
- The fact checker examines an instance (or spreader or cheerleader) of the claim, which is modeled inside itemReviewed. Note that the instance being examined is often not the original instance of the claim, but rather the most notable person who's making the claim.

Assuming this is our common understanding, we have two options to model as many instances of the claim as possible:

1. Use itemReviewed to model all the instances, i.e., one itemReviewed for each spreader of the claim. This is a much bigger burden on the reporter since itemReviewed is very heavy: we need author, publication date, etc., it seems to be a non-starter.

2. Use a simpler structure, i.e., the proposed claimUrl and claimUrlOriginal, to capture the spreaders of the claim. Assuming there is just one origin, claimUrlOriginal is singleton and claimUrl is vector. The semantics here are very different from the url attribute directly under ClaimReview since these are not URLs of the fact check.

(We do note that claimUrl is not use to capture merely reporting pieces. For example, if XYZ said 'earth is flat' and publisher A writes an article saying 'XYZ says earth is flat,' A's article shouldn't be modeled as claimUrl.)

At a high level, claimReviewed, claimUrl, claimUrilOriginal are all attributes of ClaimReview, it seems to be pretty well organized as a unit.

Dan, what do you think? We want to get the definition into the pending schema as soon as possible, so the journalists can start using it.

**Comment by danbri** [comment]
Thanks, that all helps me understand where you're coming from. 

By indirectly, what I mean is that we have made the *review* of the claim the focus, and there is no type in schema.org that actually represents the claim itself. We have types for the factcheck reviews, the people, documents, etc but we haven't yet introduced an item (that can have relationships to and from other items) representing the Claim. Maybe we don't need to.

> This is a much bigger burden on the reporter since itemReviewed is very heavy: we need author, publication date, etc., it seems to be a non-starter.

I'd put it slightly differently. None of these fields are mandatory in schema.org, sites can say as much or as little as they like. But by handling the itemReviewed as the occurrence of the claim, not the idea itself, yes it does potentially come with more baggage. Which was why I'm (tentatively) floating the possibility that we might want to have a type like "Claim" that could just represent the idea being checked, not the article or tv show or whatever where it showed up. Let's not talk about this for ages but I did want a little discussion on this point before we move to get the properties into Pending. How about we aim at having new terms in Pending by the end of next week, unless we cook up a better idea here in the meantime?

**Comment by cong-yu** [comment]
Sounds good, Dan, let's wait till end of next week in case someone comes up with a good idea.

Agreed, ClaimReview is review focused. I am a bit hesitant though to create a Claim type since it might be very confusing to people with itemReviewed also representing (an instance of) the claim. I think the rationale for keeping claimReviewed, claimUrl, claimUrlOriginal under ClaimReview directly is that they are all identified/extracted by the fact checker, not the claim maker, so it belongs more to the ClaimReview.

**Comment by danbri** [comment]
That makes sense, @cong-yu 

@rvguha @vholland - any thoughts?

**Comment by vholland** [comment]
We are adding more and more properties about the claim, including the URLs. It would be better to create a proper class for Claims and put the properties on the class rather than shoehorning into ClaimReview.

**Comment by rvguha** [comment]
I agree with Vicki. Claim should be a first class object. It does add one
level of indirection, but that could be super useful.

There are very long running claims (e.g., Birther, Gardasil, etc.) for
which someone can assign stable identifiers and once we have enough data,
build classifiers, etc.

guha

**Comment by cong-yu** [comment]
The itemReviewed within ClaimReview is in fact the claim, or more accurately, an instance of the claim. If we introduce a new Claim type, are we going to cause more confusion among the journalists in terms of the relationship between the two? Or are we talking about replacing itemReviewed with claimReviewed? (I am worried about backwards compatibility with the existing markups here ...)

**Comment by danbri** [comment]
@cong-yu my recommendation would be for reviews to stick with itemReviewed, regardless of what exactly the review is of. We already are getting reasonable questions about how to handle things like TV clips / videos, where there are two "candidates" for the item reviewed: the specific clip versus the larger material (e.g. 20 seconds vs 1 hour). Naturally it is good to have both documented, and for there to be a well known way of associating them with each other (and directly or indirectly, with the review).  The TV discussion (instigated by the Archive.org folk, who I believe are also in touch with @BillAdairDuke) is in https://github.com/schemaorg/schemaorg/issues/1686 

For the TV clip case, here is the draft approach I've sketched: https://gist.github.com/danbri/96d6265756577e5f21ad4141f053b76d ...  Here is a cut down subset to show the structure:

```javascript
{
    "@context": "http://schema.org",
    "@type": "ClaimReview",
    "datePublished": "2017-07-06",
    "url": "http://danbri.org/2017/TODO",
    "claimReviewed": "In the middle of the Cold War, the United States played a role in the overthrow of a democratically-elected Iranian government.",
    "reviewBody": "This claim is true. The UK also played a role.",
    "itemReviewed": {
        "@type": "Clip",
        "startTime": "350",
        "endTime": "370",
        "duration": "20",
        "isPartOf": {
            "@type": "VideoObject",
            "url": "https://www.youtube.com/watch?v=B_889oBKkNU",
            "name": "Clip from President Obama Speech to Muslim World in Cairo",
            "transcript": "http://www.nytimes.com/2009/06/04/us/politics/04obama.text.html"
        }
    }
  }
```

In this case, we're saying "there's a ClaimReview, and its itemReviewed is a Clip, which isPartOf a VideoObject".  If we explore the push from Vicki and Guha towards adding a Claim type (and which I agree will be very useful eventually), then we have some similar choices to make. Two analogous structures would be as follows. One has a review-claim-work structure, the other is review-work-claim.





Variant 1: "the review's itemReviewed is a Claim, which has a claimedIn of the materials carrying the claim occurence." (review-claim-work)

```javascript
{
    "@context": "http://schema.org",
    "@type": "ClaimReview",
    "datePublished": "2017-07-06",
    "url": "http://danbri.org/2018/TODO",
    "claimReviewed": "The moon is composed largely of green cheese (maybe with some patches of Camembert).",
    "reviewBody": "This claim is false, people have been there to check.",
    "itemReviewed": {
        "@type": "Claim",
        "The moon is made of green cheese.",
        "url": "https://en.wikipedia.org/wiki/The_Moon_is_made_of_green_cheese",
        "claimedIn": {
            "@type": "VideoObject",
            "url": "https://www.youtube.com/watch?v=B_889oBKkNU",
            "name": "LUNA LACTOSE NEWS: 10 Green cheese facts the MSM don't want you to know"            
        }
    }
  }
```

This is however a bit awkward, since the more canonicalized/abstracted claim is sandwiched between the two items (the review and the VideoObject or Article etc) which describe its occurence. This is fine for a single fact check but gets messy if you merge descriptions of multiple occurences since there is then no association from review to the specific article being reviewed. 

Variant 2: "the review's itemReviewed is a CreativeWork that carries an occurence of a Claim, indicated via embodiesClaim." (review-work-claim)

```javascript
{
    "@context": "http://schema.org",
    "@type": "ClaimReview",
    "datePublished": "2017-07-06",
    "url": "http://danbri.org/2018/TODO",
    "claimReviewed": "The moon is composed largely of green cheese, with some patches of Camembert.",
    "reviewBody": "This claim is false, people have been there to check.",
    "itemReviewed": {
            "@type": "VideoObject",
            "url": "https://www.youtube.com/watch?v=B_889oBKkNU",
            "name": "LUNA LACTOSE NEWS: 10 Green cheese facts the MSM don't want you to know"            
            "embodiesClaim":  {
              "@type": "Claim",
              "url": "https://en.wikipedia.org/wiki/The_Moon_is_made_of_green_cheese",
              "The moon is made of green cheese."
            }

    }
  }
```

If we continue down this route we will need to consider whether the Claim here is the exact claim represents in some NewsArticle/VideoObject etc or represents the more general idea that is likely surfacing in many places. For now I've assumed it is somewhat abstracted, but that urls might be available for very well known claims  - either at Wikipedia, or aggregations from the fact checking world, initiatives like DebateGraph etc. 

*However* If each occurrence of the Claim type really just represents a specific occurrence of a claim in a particular work, then variant one above could also be viable.

I haven't tried to mix together the TV Clip situation with the "model Claim explicitly" approach yet, but wanted to mention it here as there are similar structural issues, namely that we have to figure out which things get modeled with itemReviewed and which with other properties.

We ought also to consider whether one ClaimReview can have multiple claims/items being reviewed, or it is better just to repeat the entire structure for every (occurrence of a) claim.

**Comment by jkosslyn** [comment]
Thanks Dan. I just chatted with Guha and now better understand the desire for a "Claim" object that would be viable on its own, separate from the enclosing ClaimReview. However, to Cong's point, it will impose substantial hardship on overstretched fact check techies if we break the existing schema. Here's a thought, building on your proposed variant #1: what if Claim extended CreativeWork, kept the existing "author" semantics that we're using live, and added a field called something like "appearance" that could be an organization, person, VideoObject, etc. Thus, fact checkers who don't want to use the new "appearance" field could keep using their existing markups with no breakage, but going forward we have a richer concept of a Claim for those who want to express additional characteristics or use them outside the context of a ClaimReview.

**Comment by cong-yu** [comment]
@danbri If introducing claimUrl and ClaimUrlOriginal is absolutely a no-go, then I think reusing itemReviewed is a good compromise. Below is an example markup that corresponds to @BillAdairDuke's example. It fits well with the additional types you are creating. It is, however, not a Claim type, and it is not clear to me how to introduce an attribute that denotes the meaning of "original" since it does not make sense for general review. It is also not necessarily backward compatible since the current implementation assumes there is just one itemReviewed and that's where the claimant is extracted.

@jkosslyn Is it possible for you to create an example markup for me to understand exactly what you mean? I don't really like the above approach and would like to see what alternative proposal is there.

```
{
  "@context": "http://schema.org",
  "@type": "ClaimReview",
  "datePublished": "2018-01-05",
  "url": "http://www.politifact.com/punditfact/statements/2018/jan/05/new-century-times/white-house-staffers-removed-job-subject-false/",
  "claimReviewed": "Breaking: FBI just raided the White House, 6 people thrown out.",
  "author":
  {
    "@type": "Organization",
    "name": "PolitiFact",
    "url": "http://www.politifact.com"
  },
  "itemReviewed": [
    {
      "@type": "CreativeWork",
      "author":
      {
        "@type": "Person",
        "name": "New Century Times",
        "jobTitle": "website",
        "sameAs": "http://newcenturytimes.com/"
      },
      "datePublished": "2017-12-30"
    },
    {
      "@type": "CreativeWork",
      "url": "http://info.nct.news/2017/12/30/breaking-fbi-just-raided-the-white-house-high-ranking-officials-thrown-out/"
    },
    {
      "@type": "CreativeWork",
      "url": "https://www.vote.us.org/memo/thread/3419/fbi-just-raided-the-white-house-6-people-thrown-out/"
    }
  ],
  "reviewRating":
  {
    "@type": "Rating",
    "ratingValue": "2",
    "bestRating": "7",
    "worstRating": "1",
    "alternateName": "False"
  }
}
```

**Comment by jkosslyn** [comment]
@cong-yu sure thing, here's what I mean. Very similar to your proposal, but with only a single itemReviewed of type "Claim," and containing several instances of "appearance" in addition to the "author" field. 
```

{
  "@context": "http://schema.org",
  "@type": "ClaimReview",
  "datePublished": "2018-01-05",
  "url": "http://www.politifact.com/punditfact/statements/2018/jan/05/new-century-times/white-house-staffers-removed-job-subject-false/",
  "claimReviewed": "Breaking: FBI just raided the White House, 6 people thrown out.",
  "author": {
    "@type": "Organization",
    "name": "PolitiFact",
    "url": "http://www.politifact.com" 
    },
  "itemReviewed": {
      "@type": "Claim",
      "text": An intelligence agency decided to fight back and sent Trump a message by escorting six staffers out of the White House for failing background investigations."
      "author": {
        "@type": "Person",
        "name": "New Century Times",
        "jobTitle": "website",
        "sameAs": "http://newcenturytimes.com/" },
      "datePublished": "2017-12-30" 
      },
      "appearance": {
      "@type": "CreativeWork",
      "url": "http://info.nct.news/2017/12/30/breaking-fbi-just-raided-the-white-house-high-ranking-officials-thrown-out/"
      "position": "first"
    },
      "appearance": {
      "@type": "CreativeWork",
      "url": "https://www.vote.us.org/memo/thread/3419/fbi-just-raided-the-white-house-6-people-thrown-out/"
    },
  "reviewRating": {
    "@type": "Rating",
    "ratingValue": "2",
    "bestRating": "7",
    "worstRating": "1",
    "alternateName": "False"
  }
}
```
The underlying logic is that a review can only really be of a single claim (otherwise how can the review come to a single rating?). If that single claim appears in multiple places, we could use the "position" field or somesuch within each appearance to indicate the **first** appearance vs follow-on appearances. 

Follow-on thought: Perhaps we don't even need a new "appearance" field at all, and the existing "workExample" field might suffice, depending on how that field is currently used in practice?

**Comment by chaals** [comment]
Disconnected thoughts:

* Having a way to describe a claim (actually, that is just a statement, and in that sense a claim review is also a claim) seems useful.
* As far as I know factchecking typically works at the level of statements. If I make a speech in which I say three things, one true, one false, and one untestable, saying "the speech is not true" is less useful than identifying which things in it are untrue.
* Many claims have been around for a long time, and get useful identifiers. This seems handy in making use of fact-checking information.

**Comment by cong-yu** [comment]
@jkosslyn thanks, I revised your code so it parsed correctly and uses workExample. I think we are getting somewhere with the great discussion. See my comment after the code.

```
{
  "@context": "http://schema.org",
  "@type": "ClaimReview",
  "datePublished": "2018-01-05",
  "url": "http://www.politifact.com/punditfact/statements/2018/jan/05/new-century-times/white-house-staffers-removed-job-subject-false/",
  "claimReviewed": "Breaking: FBI just raided the White House, 6 people thrown out.",
  "author": {
    "@type": "Organization",
    "name": "PolitiFact",
    "url": "http://www.politifact.com" 
  },
  "itemReviewed": {
    "@type": ["Claim", "CreativeWork"],
    "text": "An intelligence agency decided to fight back and sent Trump a message by escorting six staffers out of the White House for failing background investigations.",
    "author": {
      "@type": "Person",
      "name": "New Century Times",
      "jobTitle": "website",
      "sameAs": "http://newcenturytimes.com/"
    },
    "datePublished": "2017-12-30" 
  },
  "workExample": [
    {
      "@type": "CreativeWork",
      "url": "http://info.nct.news/2017/12/30/breaking-fbi-just-raided-the-white-house-high-ranking-officials-thrown-out/",
      "position": "first"
    },
    {
      "@type": "CreativeWork",
      "url": "https://www.vote.us.org/memo/thread/3419/fbi-just-raided-the-white-house-6-people-thrown-out/"
    }
  ],
  "reviewRating": {
    "@type": "Rating",
    "ratingValue": "2",
    "bestRating": "7",
    "worstRating": "1",
    "alternateName": "False"
  }
}
```

Given all the proposals, let's take a look at the following scenario.

Say for example, Publisher A originally publishes an article AA spreading a rumor. Politician X notices that article and spreads the same rumor in one of his speeches. Publisher B reports the speech in an article BB, which also spreads the rumor. Fact checker C comes in and fact checks X and writes the fact checking article.

In the less semantically rich first proposal (claimUrl, claimUrlOriginal), the itemReviewed will be about X making the speech, which links to the official speech site, while claimUrl and claimUrilOriginal link to BB and AA, respectively. This works because the itemReviewed, claimUrl, claimUrlOriginal are equivalent, all are just instances of the claim.

In the more semantically rich current proposal with workExample, the one stored inside itemReviewed gets the elevated status because it is the root element containing all the workExamples. Thus, in this case, should the speech be in the itemReviewed or the article AA? If the former, the semantics is a bit weird since politician B is not the original claim maker. If the latter, what about cases where the original claim maker can not be identified?

So this may be at the root of the discussion: if we want to capture Claim as @chaals and many on this thread are aiming for, what fields really identify a Claim? So far, we have pretty good fields to identify instances of a Claim, but not really a Claim in the sense that can unify all the instances. Is it possible that we don't actually have that information for most cases?

**Comment by jkosslyn** [comment]
@cong-yu that proposal LGTM, @danbri what do you think?

@cong-yu in terms of the scenario you pose, the claim would be the rumor as asserted by Publisher A in article AA, and as amplified by Politician X and Publisher B. If the amplifications cite the original claim by Publisher A, then they would be workExamples; if the amplifications don't cite AA but rather assert the claim as fact, then they could be considered new examples of the claim with new authorship by Politician X and Publisher B. 

In terms of the general principle, I think you're right that some Claims will have more fields and some will have less; some will be unified and some may be disjoint; etc. This seems to be a fundamental, unavoidable fact of the messiness of the real world. If there are enough instances of Claims with overlapping fields, perhaps they could be programmatically merged, at least in some cases.

**Comment by danbri** [comment]
This direction is looking good, but let me check something. Good idea to reuse workExample, but is it intended that it is attached to the ClaimReview not the Claim (which was where "appearance" was shown in Justin's original).
This is the most recent example from Cong: 
https://search.google.com/structured-data/testing-tool/u/0/?url=https%3A%2F%2Fgist.githubusercontent.com%2Fdanbri%2F07c8581de424942c446c4fa70a76f1ee%2Fraw%2F3aa786d592783d3020ac2a53b917933cc819d50c%2Fgistfile1.txt#url=https%3A%2F%2Fgist.githubusercontent.com%2Fdanbri%2F07c8581de424942c446c4fa70a76f1ee%2Fraw%2F3aa786d592783d3020ac2a53b917933cc819d50c%2Fgistfile1.txt

**Comment by cong-yu** [comment]
@danbri Good call, that was a typo, fixed.

@jkosslyn a bit confused here, are you suggesting the following: If the instances of the claim are independent (i.e., no citation connection), we should model them as separate itemReviewed; if they form a citation chain, we model them as workExamples where the one everyone cites becomes the itemReviewed? That seems too complicated semantically to maintain and also not backward compatible. Perhaps we can assume within a single ClaimReview, the instances of the claim are always connected and the reporter can simply use their judgement to pick one that gets the itemReview's author/datePublished treatment?

**Comment by danbri** [comment]
@cong-yu @jkosslyn having a single itemReviewed seems a little simpler conceptually to me, and consistent with the way other kinds of reviews are handled. But it can be awkward (as it was with the TV video vs smaller clip example) since you have to somewhat arbitrarily pick one entity as the focal point, even if the others can also be associated via relationships.

I would like to leave open the possibility (beyond the fact check review usecase) of having Claims relate to Claims, eg. well known claims having well known URLs. I'm not sure if there is a need for Claim-to-Claim metadata within typical fact check claim reviews though. We can probably get away with using sameAs URLs for the well known links, to keep the review markup simple.

Here's an example I've had in mind for a while, https://debategraph.org/Stream.aspx?nid=33108&vt=rgraph&dc=focus - debategraph's representation of the (claim that) "the 1953 coup deposed the democratically elected Mosaddeq government".  Actually that is a pretty mild claim, the stronger claim is the more specific one that CIA and MI5 were the initiators of the coup. By having an explicit representation of these (increasingly detailed) claims, decoupled from any specific fact check review, we get an interesting foundation for linking this kind of data together.

**Comment by jkosslyn** [comment]
@danbri agreed, having a single itemReviewed is conceptually clearer to me as well. I noticed that Clip inherits from CreativeWork; could we allow the itemReviewed to be any subclass of CreativeWork? As long as it has a url field, adding more optional fields wouldn't hurt the core usecase. 

@cong-yu If the instances of the claim are independent (i.e., no citation connection), I think we should model them as separate ClaimReviews. I.e. two independent claims with two independent sources and contexts means two independent reviews. After all, in fact checking we've seen so many cases where the context tips the scales of the review. 

Of course there may always be corner cases where citations are not explicit but are implied; in those cases editorial judgement would make the call.

**Comment by danbri** [comment]
@jkosslyn - for a ClaimReview it may make sense to encourage only CreativeWork. There are other kinds of reviews too (cars, hotels, employers, ...), but they at least can share the common structure of having a single focal itemReviewed.


In passing (and wary of distractions), we should also note for the record that there may be some commonalities between Claim(Review) markup and Question/Answer sites. That does not mean we need to converge the two forms of markup, but it is worth bearing in mind that e.g. matters of historical or scientific fact are often debated on sites that are structured in terms of questions and candidate answers. For example: https://history.stackexchange.com/questions/26495/how-secret-was-the-us-and-british-involvement-in-the-1953-iranian-coup-d%C3%A9tat?rq=1 (see [data extracted by Google](https://search.google.com/structured-data/testing-tool?url=https://history.stackexchange.com/questions/26495/how-secret-was-the-us-and-british-involvement-in-the-1953-iranian-coup-d%C3%A9tat?rq=1#url=https%3A%2F%2Fhistory.stackexchange.com%2Fquestions%2F26495%2Fhow-secret-was-the-us-and-british-involvement-in-the-1953-iranian-coup-d%C3%A9tat%3Frq%3D1)).

**Comment by cong-yu** [comment]
OK, I think we are getting to some consensus. The remaining question is that, given we are not introducing new field, do we really need a new Claim type?

I do think there is value in introducing Claim type on its own, independent of the ClaimReview, with all the benefits of linking everything together, but it does sound like beyond the scope of this current extension to ClaimReview?

**Comment by danbri** [comment]
@cong-yu isn't the ability to say what we need without adding more custom properties a feature not a bug?

**Comment by cong-yu** [comment]
@danbri do you mean we can introduce the Claim type without needing to add a new field?

**Comment by danbri** [comment]
We could.

**Comment by rvguha** [comment]
Yes, we can add the new type (Claim) without a new property/field.

a ClaimReview could either have a particular article as its itemReviewed or
a Claim.

e.g., consider the claim that Lee Harvey Oswald did not act alone. This is
a claim that has been restated in thousands of articles. Rather than
picking one of these articles to be the itemReviewed, it could be the
abstract Claim.

This construct could be useful for  science, medical and other fact
checkers who don't want to point to some particular publication and say
that it is false, but rather, more abstractly, provide evidence for or
against some claim/theory.

guha

**Comment by thadguidry** [comment]
Claim:  Guha is intelligent.  All humans have intelligence.  Sometimes :)

+1 for new Claim type.

**Comment by cong-yu** [comment]
@rvguha @danbri Sounds good, in terms of backward compatibility implementation, can we have the itemReviewed be of both Claim and CreativeWork type?

**Comment by rvguha** [comment]
Claim will be a subclass of CreativeWork.

guha

**Comment by danbri** [comment]
We should add guidance on common patterns to the ClaimReview textual documentation; it's worth mentioning the Clip vs VideoObject scenario there too.

**Comment by BillAdairDuke** [comment]
Thanks to all of you for the excellent discussion. I'm reading this from the perspective of a journalist, so I'm not sure I've understood all the comments. But I thought it would be helpful to add my perspective since I'm sort of speaking for all the fact-checkers that will have to fill out the schema.

The reclassification of claims sounds fine and probably helpful. I agree with Justin's points that anything new should be backwards compatible with previous versions. I also would urge you to use labels for fields that journalists will understand intuitively. Part of the reason I proposed these changes was the confusion by me and others about fields that weren't clear to journalists. 

Would it be possible for one of you to summarize the consensus so far and what it would mean for the journalists filling out the schema either in their own CMS or on the Share the Facts widget?

Thanks!

Bill

**Comment by danbri** [comment]
Absolutely agree with Bill that usability (including of our documentation/examples, not just the abstract structure is critical. Justin's summary overall makes sense, the only place I feel wobbly is the "position": "first" construction. How important is this? Could we just encourage use of dates (on cCaims, ClaimReviews, and CreativeWorks)?

*yes, subtypes of CreativeWork are also expected; sometimes a CreativeWork may be further contextualized by also describing a larger work it is part of too, eg video clip, book chapter, blog post.

**Comment by cong-yu** [comment]
@BillAdairDuke Bill, I think here is the consensus code among the thread.

```
{
  "@context": "http://schema.org",
  "@type": "ClaimReview",
  "datePublished": "2018-01-05",
  "url": "http://www.politifact.com/punditfact/statements/2018/jan/05/new-century-times/white-house-staffers-removed-job-subject-false/",
  "claimReviewed": "Breaking: FBI just raided the White House, 6 people thrown out.",
  "author": {
    "@type": "Organization",
    "name": "PolitiFact",
    "url": "http://www.politifact.com" 
  },
  "itemReviewed": {
    "@type": ["Claim", "CreativeWork"],
    "text": "An intelligence agency decided to fight back and sent Trump a message by escorting six staffers out of the White House for failing background investigations.",
    "author": {
      "@type": "Person",
      "name": "New Century Times",
      "jobTitle": "website",
      "sameAs": "http://newcenturytimes.com/"
    },
    "datePublished": "2017-12-30",
    "workExample": [
      {
        "@type": "CreativeWork",
        "url": "http://info.nct.news/2017/12/30/breaking-fbi-just-raided-the-white-house-high-ranking-officials-thrown-out/",
        "position": "first"
      },
      {
        "@type": "CreativeWork",
        "url": "https://www.vote.us.org/memo/thread/3419/fbi-just-raided-the-white-house-6-people-thrown-out/"
      }
    ]
  },
  "reviewRating": {
    "@type": "Rating",
    "ratingValue": "2",
    "bestRating": "7",
    "worstRating": "1",
    "alternateName": "False"
  }
}
```

A few more notes:

1. @jkosslyn For backward implementation compatibility, let's keep both Claim and CreativeWork as a type of the itemReviewed. In the future, this may not be necessary, but I want it to be on the safe side.

2. @danbri datePublished does not provide semantics of "original," since even if this example is earliest among the examples, it does not necessary mean it is the original claim.  ["position": "first"] solves that.

3. @BillAdairDuke workExample is an existing field and thus preferred. If the journalists strongly prefers some other terms, then we can introduce a new field for the Claim type. At the same time, the widget can use any term and seamlessly map into this field.

4. While the structure has a hierarchy, itemReviewed to workExample, the semantics is not really hierarchical, journalists should feel free to put whatever they consider as the main claim-maker into the root structured.

5. The Claim type will allow for future extension, including clips, books, etc. and eventually provide the basis for a Claim database.

@BillAdairDuke Do you want Chris to take a look at the code? I am also happy to chat for more clarification if needed.

**Comment by danbri** [comment]
"the semantics is not really hierarchical" - unfortunately that is the exact problem with position: first, ... since the properties of the work being described aren't contextualized to the Claim or ClaimReview, but have to be objective properties of that entity directly.

One way to do that would be to use a more specific property than workExample. We might for example define "appearance" and also "firstAppearance" as relations linking the Claim to the CreativeWork. That would address my concern about position:first and provide a way to name the link in a journalist-friendly way while still having a documented pattern based on workExample. I think! Typed from an airport on a phone, I may have missed some nuance.

**Comment by cong-yu** [comment]
@danbri hmm ... isn't it the intension of the position field to put the item in context? (From the description: the position of an item in a series or sequence of items.)  But I am open to introduce a new field or two (originalAppearance and appearance) under Claim. Shall we just go ahead and do it?

@thadguidry The journalist may not know who made the original claim, and thus it is not among the examples.

**Comment by danbri** [comment]
@cong-yu I have made a pass at implementing this. It needs more attention including an example  or two and improvements to the text for the existing ClaimReview and itemReviewed definitions. But there are rough cut definitions for Claim, appearance and firstAppearance. Wording can be refined. You can see the raw definitions in the links to the configuration files here, or staged at http://pending.webschemas.org/Claim

**Comment by thadguidry** [comment]
@danbri Let's try to avoid using the type name again in the definition (why do you and @RichardWallis  always seem to do that? :-) ) Suggest changing this "a specific, factually-oriented claim ..."  to this "a specific, factually-oriented **statement** ..." or another synonym of choice.

**Comment by cong-yu** [comment]
Thanks @danbri !

A couple comments on the pending Claim type:

1. Shall we clarify a bit on the text property for Claim and the claimReviewed property for ClaimReview? If a Claim is attached to a ClaimReview, then the text field, if missing, is assumed to the same as the the claimReviewed property of the ClaimReview it is attached to

2. For firstAppearance: shall we consider change "first known" to "original", or better, change firstAppearance to originalAppearance?  This may be a question for @BillAdairDuke as in do we care if this field contains the first known instance of the claim even though the reporter knows for sure this is not the origin?

**Comment by subbuvincent** [comment]
Hi folks, I just caught up with this thread. All three central points are valid. 
- the need from the use case @BillAdairDuke brought up for claimOriginal and claimUrl
- and the opportunity for a class of <abstract> claims, a-la canonicals for claims @vholland @rvguha 
- and the reality of maintaining backwards compatibiity that @cong-yu rightly brings up 

I tried something here using @danbri exploration and still constraining with the original use case @BillAdairDuke wants to address. Perhaps we could create @type Claim in a forward looking way, and let claimReviewed take both text values and type-Claim values. The rest can follow. 

```
{
    "@context": "http://schema.org",
    "@type": "ClaimReview",
    "datePublished": "2017-07-06",
    "url": "http://danbri.org/2018/TODO",
    "claimReviewed": [“The moon is composed largely of green cheese, with some patches of Camembert.",
     		      {
		      “@type": "Claim",
              	      “claimStage”: “Original”,
                      "url": "https://en.wikipedia.org/wiki/The_Moon_is_made_of_green_cheese",
              	      “text”: ”The moon is made of green cheese."
            	      },
     		      {
		      “@type": "Claim",
              	      “claimStage”: “Repeated”,
                      "url": "https://the-moon-is-made-of-green-cheese.com”,
              	      “text”: ”The moon is made of green cheese."
            	      }
 		      ],
    "reviewBody": "This claim is false, people have been there to check.",
    "itemReviewed": {
	   	     "@type": "CreativeWork",
    		     "author":
    				{
      				"@type": “Organization”,   // just changed Person to Org - my pref.
      				"name": "New Century Times",
      				"sameAs": "http://newcenturytimes.com/"
    				},
    		    "datePublished": "2017-12-30"
  		    },
   "reviewRating":
    {
    "@type": "Rating",
    "ratingValue": "2",
    "bestRating": "7",
    "worstRating": "1",
    "alternateName": "False"
  }
}
```
Does this help?

**Comment by cong-yu** [comment]
@subbuvincent Thanks Subbu, I do believe the mixture of textual value and nested value will break current implementations. If it is either/or, it might be tricky for the existing parsers.

**Comment by subbuvincent** [comment]
@cong-yu Thanks. And this is sheepish but it looks like when I posted my comment, my browser was showing a cached version so I had missed several comments going back 4 days! Never mind :). It's great to see more progress since then.

**Comment by vinnygreen** [comment]
Hello all! I'm head of operations and development for Snopes.com. I integrated the claim review mark-up into Snopes' WordPress Theme. I spent a lot of time thinking about this markup because it relates to the other projects I am working on like a misinformation triage engine and a commercial API. 

The ClaimReview process is built into our fact-check template with a simple UI for our staff. My staff never sees the schema; they fill in fields I labeled in a way they understand.

I've reviewed the thread and wanted to chime in with Snopes' position on the markup:

- SameAs field and itemReviewed are confusing, but the simple solution is just allowing multiple itemReviewed for each distinct claim which is not currently permitted. 
 - If a page has many distinct claims, it requires many claimReviews. If a claimReview has multiple sources, it should require multiple itemRevieweds.
 - With multiple itemReviews available, we can have multiple sources of information and use the sameAs field as intended. 
- We could instruct end users to assume the earliest date of the itemReviewed is the earliest source of that claim that the publisher has identified. If they wanted to know what the individual fact checker considered to be the earliest claim URL, they would find the oldest datePublished in one of the many itemRevieweds and then search for the sameAs URL that links to an article. 
-Alternatively, we could add a new URL field called `claimUrl` in the itemReviewed section for the exact URL where the claim lives which could have its own datePublished as well.
-My concern is that new or ambiguous fields will be ignored or misinterpreted by end-users creating more misinformation. Not to mention the burden on the journalist curating the data that can be inferred via a query by the end-user. The more data entry work for the journalist the greater the chance for error. 
-Backward compatibility is incredibly important. 

I know I jumped in very late, but I hope this is helpful. I can be reached directly at vinny@snopes.com 

**Snopes Ideal Schema**
```{
  "@context": "http://schema.org",
  "@type": "ClaimReview",
  "datePublished": "2016-06-22",
  "url": "http://snopes.com/fact-check/a-snopes-fact-check/",

  "itemReviewed":
  {
    "@type": "CreativeWork",
    "author":
    {
      "@type": "Organization",
      "name": "Bad Actor 1",
      "sameAs": "https://twitter.com/badactor1",
      "sameAs": "https://wikipedia.com/badactor1",
    },
    {
      "claimUrl": "https://badactor1blog.com/article-with-claim/",
      "datePublished": "2016-06-20"
   }
  },

  "itemReviewed":
  {
    "@type": "CreativeWork",
    "author":
    {
      "@type": "Organization",
      "name": "Bad Actor 2",
      "sameAs": "https://twitter.com/badactor 2",
      "sameAs": "https://wikipedia.com/badactor 2",
      "claimUrl": "https://badactorblog.com/article-with-claim/"
    },
    {
      "claimUrl": "https://badactor2blog.com/article-with-claim/",
      "datePublished": "2016-06-20"
   },
    {
      "claimUrl": "https://facebook.com/badactor2blog/212131342334536342",
      "datePublished": "2016-06-20"
   }
  },

  "claimReviewed": "Bad Actors say what?",
  "author":
  {
    "@type": "Organization",
    "name": "Snopes.com"
  },
  "reviewRating":
  {
    "@type": "Rating",
    "ratingValue": "1",
    "bestRating": "5",
    "worstRating": "1",
    "alternateName" : "False"
  }
}```

**Comment by danbri** [comment]
I've been looking into the consequences of this for neighbouring pieces of Schema.org, i.e. non fact-checking reviews. I'll open an independent issue on that, since Schema.org needs to have a clear picture of when/whether itemReviewed can be repeated. I believe we can make this work and just need to do the housekeeping to keep related definitions in sync.

I'd like to say

* "itemReviewed" in general case: most typically points to a single thing
* but can point to several things, if a review is reviewing several things
* conventions and common patterns may vary between the different subtypes of Review (UserReview, CriticReview, ClaimReview etc.).
* Fact checking reviews (reviews of Claims, i.e. ClaimReview) can use repeated itemReviewed to describe various appearances of several claims that it makes sense to evaluate within a common review (i.e. fact-check). We need not be strict here about whether they are multiple appearances of exactly the same claim, or variations on a theme.
* Other kinds of reviews sometimes have multiple items reviewed in an integrated writeup, this is probably more typical of critic reviews than user reviews, and what we do here for fact checking shouldn't impact those other types, beyond the general clarification that "itemReviewed" is sometimes repeated.
* We can note that it is possible to use the various properties of the items reviewed (their publication dates, or perhaps mainEntityOfPage markup) to point to the earliest or primary item, if there are several. Note that we are not imposing a single approach to that matter here. Conventions may emerge, and they may be subtype-specific.

**Comment by cong-yu** [comment]
FYI The following is the example code taking Vinny and Dan's comments into consideration. It leverages the repeatability of itemReviewed, which Dan is hoping to get approved soon. No new type or attribute is introduced (Claim type is already being proposed in a separate issue).

Regarding the granularity of the repeatability:

1. Same claim, same claimant, different instances => use different workExample within same itemReviewed for repetition

2. Same claim, different claimant => use itemReviewed for repetitiion

3. Different claim => use different ClaimReview

Vinny has a good point on being careful about the variants of the claim (whether two claims are really different or simply slight variants of the same), it is an editorial call and the idea is that each publisher should think carefully about that and use either 3 or 1/2 as appropriate.

`
{
  "@context": "http://schema.org",
  "@type": "ClaimReview",
  "datePublished": "2018-01-05",
  "url": "http://www.politifact.com/.../white-house-staffers-removed-job-subject-false/",
  "claimReviewed": "Breaking: FBI just raided the White House, 6 people thrown out.",
  "author": {
    "@type": "Organization",
    "name": "PolitiFact",
    "url": "http://www.politifact.com" 
  },
  "itemReviewed": [
  {
    "@type": ["Claim", "CreativeWork"],
    "author": {
      "@type": "Organization",
      "name": "New Century Times",
      "sameAs": "http://newcenturytimes.com/"
    },
    "workExample": [
      {
        "@type": "CreativeWork",
        "url": "http://info.nct.news/...-white-house-high-ranking-officials-thrown-out/",
        "datePublished": "2017-12-30"
      },
      {
        "@type": "CreativeWork",
        "url": "http://info.nct.news/...-white-house-high-ranking-officials-thrown-out-2/",
        "datePublished": "2017-12-31"
      }
    ]
  },
  {
    "@type": ["Claim", "CreativeWork"],
    "author": {
      "@type": "Organization",
      "name": "Vote US",
      "sameAs": "http://www.vote.us.org/"
    },
    "workExample": [
      {
        "@type": "CreativeWork",
        "url": "https://www.vote.us.org/...-raided-the-white-house-6-people-thrown-out/",
        "datePublished": "2017-02-16"
      }
    ]
  }],
  "reviewRating": {
    "@type": "Rating",
    "ratingValue": "2",
    "bestRating": "7",
    "worstRating": "1",
    "alternateName": "False"
  }
}
`

**Comment by danbri** [comment]
Thanks @cong-yu - we already added Claim (with the usual 'pending' status for new things) in this last release. 

Wouldn't we rather use the "appearance" (and even "firstAppearance") properties to link from the Claim to the CreativeWork? (rather than workExample)

There's also no strong need to multiple-type ("@type": ["Claim", "CreativeWork"]); Claim is a defined in the schemas as a CreativeWork so the extra info doesn't add much except page weight.

**Comment by BillAdairDuke** [comment]
One variation we often see in political fact-checking is similar but not identical claims. For those the fact-checkers would have to judge if they are similar enough to count them as the same.

So would that be a be a fourth variation in your options, Cong?

4. Similar claim => use different ClaimReview (but somehow designate its similarity?)

**Comment by danbri** [comment]
On 16 July 2018 at 13:30, BillAdairDuke <notifications@github.com> wrote:

> One variation we often see in political fact-checking is similar but not
> identical claims. For those the fact-checkers would have to judge if they
> are similar enough to count them as the same.
>
> So would that be a be a fourth variation in your options, Cong?
>
> 4. Similar claim => use different ClaimReview (but somehow designate its
> similarity?)
>

Now that we have a type for Claim, it does open up some options in this
area. We have to be careful not to make things too complex, but one idea
would be to use relationships between several Claims. This could be done as
conventions on top of the basic pattern described here. In the markup this
could be either through nesting or through assigning simple IDs to each
Claim, so that we could point from one claim to another.

For example, if the reported public speech was in some way ambiguous,
perhaps a context which was open to multiple interpretations (e.g. vague
geographic or temporal aspects), you might want to write down several forms
of the Claim. Similarly with translations (we already
have translationOfWork and translator properties for general use). This
could be used either in the case of fact checks written for a
multi-language audience, or when the source language differs from that in
which the fact check is being written (e.g. western writing about Israel,
Iran political speech).

I wouldn't want to get too far into these sorts of cases without staying
close to real world examples. Bill - could you share any that you think
give a good illustration to guide us here?

This could potentially work either by having several ClaimReviews, or by
repeating itemReviewed in simpler cases, depending on where we draw the
line. It would be good to have some clear wording for this, i.e. for when
to break out a different ClaimReview. My feeling is that if broadly the
same family of specific claims by the same speaker is being discussed, a
single review may be natural. But there will be lots of times when it'll be
more natural to have several reviews.

**Comment by cong-yu** [comment]
@danbri yep, appearance should work as well, I will update my comment

@BillAdairDuke Yeah, I would leave it to the fact checkers to determine how similar two claims are to each other that a single ClaimReview is warranted ... @danbri this is something hard to concretely define since it is somewhat subject, and I would not try to clarify this too much

**Comment by cong-yu** [comment]
@danbri FYI using just Claim still gives error in SDTT so I will keep both for now.

**Comment by danbri** [comment]
@cong-yu - then let's update the Google tool! 

And yes, I agree re hard to define. Maybe something soft like:

>  "several variations on the same claim (e.g. different phrasings but in the same social/political context) can be handled within a single ClaimReview via repetition of itemReviewed, but the idea of a ClaimReview is that more or less a single idea is being examined."

**Comment by BillAdairDuke** [comment]
How about this:

Mitt Romney:

Says Barack Obama began his presidency "with an apology tour."
https://www.politifact.com/truth-o-meter/statements/2012/oct/17/mitt-romney/mitt-romney-says-barack-obama-began/

Several times, Obama "has apologized for what he deems to be American arrogance, dismissiveness, and derision; for dictating solutions, for acting unilaterally ... ."
https://www.politifact.com/truth-o-meter/statements/2010/mar/15/mitt-romney/obama-remarks-never-true-apology/

President Obama "went around the world and apologized for America."
https://www.politifact.com/truth-o-meter/statements/2011/sep/22/mitt-romney/mitt-romney-repeats-claim-obama-went-around-world-/

**Comment by danbri** [comment]
Here's some text towards documenting how all these constructs can fit together...


<h3 id="keyconcepts"><code>Key Concepts</code></h3>
<p>ClaimReview-based factcheck markup defines a structure that corresponds to the kind of information included in many fact-checking pages.</p>

* the fundamental notion is a <a href="https://schema.org/ClaimReview">ClaimReview</a>. This has an <a href="https://schema.org/author">author</a>, which is typically an <a href="https://schema.org/Organization">Organization</a> (i.e. the fact checking organization or publisher), but could also be a <a href="https://schema.org/Person">Person</a>.
* The <a href="https://schema.org/claimReviewed">claimReviewed</a> property of a <a href="https://schema.org/ClaimReview">ClaimReview</a> summarizes the claim being reviewed. This may include clarifications of the original wording to address intelligibility, civility, context or brevity, and can include translations.
* This value of the <a href="https://schema.org/claimReviewed">claimReviewed</a> property is typically a simple textual string (but could be a <a href="https://schema.org/Claim">Claim</a> with a <a href="https://schema.org/text">text</a> property, although this is not encouraged).
* The <a href="https://schema.org/itemReviewed">itemReviewed</a> property of <a href="https://schema.org/ClaimReview">ClaimReview</a> indicates specific manifestations of the claim being reviewed. This can either be a <a href="https://schema.org/Claim">Claim</a> [preferred] or [historically] a <a href="https://schema.org/CreativeWork">CreativeWork</a> within which the claim is described or reported.
* The value of <a href="https://schema.org/itemReviewed">itemReviewed</a> (preferably a <a href="https://schema.org/Claim">Claim</a> to avoid ambiguity) has an <a href="https://schema.org/author">author</a>, which is a <a href="https://schema.org/Person">Person</a> or <a href="https://schema.org/Organization">Organization</a> that has made the claim.
* A <a href="https://schema.org/Claim">Claim</a> can be associated with a <a href="https://schema.org/CreativeWork">CreativeWork</a> it occurs in, using the <a href="https://schema.org/appearance">appearance</a> or <a href="https://schema.org/firstAppearance">firstAppearance</a> properties. This is preferable to describing appearances using <a href="https://schema.org/itemReviewed">itemReviewed</a> as it distinguishes more explicitly between the <a href="https://schema.org/author">author</a> of the <a href="https://schema.org/Claim">Claim</a> versus <a href="https://schema.org/author">author</a> of materials discussing those claims.</li>
* The <a href="https://schema.org/reviewRating">reviewRating</a> property of the <a href="https://schema.org/ClaimReview">ClaimReview</a> indicates a <a href="https://schema.org/Rating">Rating</a> of the claim.
* A rating can be summarized textually with a <a href="https://schema.org/alternateName">alternateName</a> property, and with a numerical rating on a scale from <a href="https://schema.org/worstValue">worstValue</a> (lowest) to <a href="https://schema.org/bestValue">bestValue</a> (highest).
* The <a href="https://schema.org/author">author</a> (or <a href="https://schema.org/creator">creator</a>, <a href="https://schema.org/publisher">publisher</a>) of a <a href="https://schema.org/ClaimReview">ClaimReview</a>, or of a <a href="https://schema.org/Claim">Claim</a>, or <a href="https://schema.org/CreativeWork">CreativeWork</a>, can be either an <a href="https://schema.org/Organization">Organization</a> or <a href="https://schema.org/Person">Person</a>.

**Comment by rvguha** [comment]
Bill, Cong,

 Can you review this?

guha

**Comment by rvguha** [comment]
+Simon Baumgartner <simonba@google.com>

**Comment by danbri** [comment]
@rvguha - Simon and I wrote this together. /cc @cong-yu 

We were originally thinking of it for Google's own documentation but it felt like a better fit for Schema.org. @tmarshbing - can you take a look from Bing's perspective too?

**Comment by sens3** [comment]
The key concepts you outline are semantically correct and make sense to me. 
My only concern is that all this flexibility makes things confusing. 

Can we run this by one or two fact checkers to see whether this makes sense to them?

**Comment by danbri** [comment]
Thanks @sens3, that's why we're here. I've pinged a few folks by email and Twitter who might not be monitoring Github closely, too...

Some of the flexibility we can nudge people away from for now (e.g. having claimReviewed be anything other than a string is not needed until there's a strong demand for multi-lingualism and clarity about translations). But a lot of it comes with the territory, as soon as the question of multiple appearances of the "same" claim - made by different parties, reported in different places - is considered.

I would love to tighten things down to a few core patterns and have examples of each, so long as we're consistent with the direction that that fact checking community is headed.

**Comment by vinnygreen** [comment]
@danbri Can you build out the an empty example schema with all the bells and whistles? I can hunt down some real examples that mirror the possible complexity and fill in the schema.

**Comment by danbri** [comment]
Hi @vinnygreen - thanks. Here's a fairly simple example @sens3 and I been talking through. Essentially the evolution is from having a CreativeWork (e.g. some news article) be the value of "itemReviewed", to having a Claim there. Then we can point from that Claim to its appearances in the press etc., using "appearance". This helps avoid ambiguity when we want to distinguish authors of articles and claims.

```
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "ClaimReview",
  "datePublished": "2016-06-22",
  "url": "http://example.com/news/science/worldisflat.html",
  "claimReviewed": "The world is flat",
  "itemReviewed": {
    "@type": "Claim",
    "author": {
      "@type": "Organization",
      "name": "Square World Society",
      "sameAs": "https://example.flatworlders.com/we-know-that-the-world-is-flat"
    },
    "datePublished": "2016-06-20",
    "appearance": {
      "@type": "NewsArticle",
      "url": "http://skeptical.example.net/news/a122121",
      "name": "Square Earth - Flat earthers for the Internet age",
      "datePublished": "2016-06-22",
      "author": {
        "@type": "Person",
        "name": "T. Tellar"
      }
    }
  },
  "author": {
    "@type": "Organization",
    "name": "Example.com science watch"
  },
  "reviewRating": {
    "@type": "Rating",
    "ratingValue": "1",
    "bestRating": "5",
    "worstRating": "1",
    "alternateName": "False"
  }
}
</script>

**Comment by cguess** [comment]
I’ll jump in and agree that ambiguity will lead to paralysis for anyone not used to relational databases and systems. Having any data nested more than one level, two at the extreme, is, from my perspective, untenable. From the journalism perspective  every field should, ideally, have one and only one right way to fill out. It’s easier for me to explain how to shoe horn something than it is to remember options.

**Comment by Fleker** [comment]
It seems, from the specification, that a `Claim` is designed to serve as a child of a `ClaimReview`. While this makes sense for fact-checking, I wonder if this could be applied in reverse. A news site may be interested in verifying their trustworthiness through the [IEEE P7011](https://site.ieee.org/sagroups-7011/) standard. As such, they may want to markup claims they make with links to a fact-checking service, either 3P (Snopes) or 1P (if the news service has their own fact checking).

So there would be an interest in modifying the order and adding a new `review` or some other link to a fact-checker. What do you think?

```
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "Claim",
  "datePublished": "2016-06-22",
  "url": "https://trustworthy.news/science/world-is-flat",
  "abstract": "The world is flat",
  **"review"**: {
    "@type": "ClaimReview",
    "datePublished": "2016-06-28",
    "url": "https://facts.news/science/world-is-flat",
    "author": {
      "@type": "Organization",
      "name": "News Facts Fact Checker"
    },
    "reviewRating": {
      "@type": "Rating",
      "ratingValue": "1",
      "bestRating": "5",
      "worstRating": "1",
      "alternateName": "False"
    }
  }
}
</script>
```

This may alternatively be scattered throughout a news piece with a handful of reviews as an extension of a link that can be link to a fact checker. Then the fact checker can provide a `ClaimReview` element that returns to the current news provider to provide verification or denial of that fact.

```
As we all know,
<a href="/previous-story-on-news-service" itemscope itemtype="http://schema.org/Claim">
    <meta itemprop="review" itemscope itemtype="http://schema.org/ClaimReview>
       <meta itemprop="url" content="https://facts.news/science/world-is-flat" />
    </meta>
    the world is flat
</a>.
```

**Comment by 0983210321** [comment]
- [ ]

**Comment by gabrielgrant** [comment]
Has there been any movement on this? @danbri 's [proposal of how this all fits togehter](https://github.com/schemaorg/schemaorg/issues/1828#issuecomment-473390948) seems well considered, and doesn't seem to have elicited any substantial objections. The [Claim type (listed as "Pending"](https://schema.org/Claim) seems useful, and, given the lack of activity here, I'm tempted to make use of it as-is, but would obviously prefer it were officially adopted. What's needed to make that happen?

One minor question re: the proposal: why is it the [claimReviewed](https://schema.org/claimReviewed) property is recommended to be a simple textual string rather than a [Claim](https://schema.org/Claim) with a [text](https://schema.org/text) property? it seems the content of the claim belongs more on the claim itself than solely in the review object. Is the suggestion that it be duplicated in both? or is there some reason that the Claim not actually contain the claim text?

**Comment by Fghj77800** [comment]
> After discussing the use of ClaimReview with fact-checkers, there is confusion among the fact-checkers about how to populate the field for SameAs field. Some fact-checkers have been using for the URL of speeches or articles where the claim appeared to provide evidence and details of the claim; others have been using it for the home page/knowledge base article on the person or entity that made the statement.
> 
> Both have value for search engines and other uses. It’s valuable to be able to see a knowledge base article on the person or entity being checked; it’s also valuable to be able to see where a false claim was published, particularly the article or speech where the claim originated.
> 
> **The solution:** We propose that: sameAs will continue to be used for the entity or speaker that made the the claim (we called it the “who to blame” field). This field would typically be the home page of the person making the claim, or their Wikipedia page.
> 
> We will add a field(s), under ClaimReview, called claimUrlOriginal of type URL to designate the URL of the origin of a claim such as a speech or article; fact-checkers could include additional URLs (claimUrl) of type URL if the claim has been repeated in other publications.
> 
> For example, here is a fact-check by PolitiFact of a claim by New Century Times. The SameAs field is the website’s URL. The claimURLOriginal field would be the article that contained the claim that was fact-checked. That claim also appeared on Vote.us.org, which is listed as a claimURL.
> 
> {
>   "@context": "http://schema.org",
>   "@type": "ClaimReview",
>   "datePublished": "2018-01-05",
>   "url": "http://www.politifact.com/punditfact/statements/2018/jan/05/new-century-times/white-house-staffers-removed-job-subject-false/",
>   "claimReviewed": "Breaking: FBI just raided the White House, 6 people thrown out.",
>   "claimUrlOriginal": "http://info.nct.news/2017/12/30/breaking-fbi-just-raided-the-white-house-high-ranking-officials-thrown-out/",
>   "claimUrl": ["https://www.vote.us.org/memo/thread/3419/fbi-just-raided-the-white-house-6-people-thrown-out/"],
>   "author":
>   {
>     "@type": "Organization",
>     "name": "PolitiFact",
>     "url": "http://www.politifact.com"
>   },
>   "itemReviewed":
>   {
>     "@type": "CreativeWork",
>     "author":
>     {
>       "@type": "Person",
>       "name": "New Century Times",
>       "jobTitle": "website",
>       "sameAs": "http://newcenturytimes.com/"
>     },
>     "datePublished": "2017-12-30"
>   },
>   "reviewRating":
>   {
>     "@type": "Rating",
>     "ratingValue": "2",
>     "bestRating": "7",
>     "worstRating": "1",
>     "alternateName": "False"
>   }
> }
> Edited 2018-01-23 by [@danbri](https://github.com/danbri) to highlight key terms, omit HTML script tag, improve formatting.
